app/assets/javascripts/service-worker.js.erb

```diff
@@ -7,11 +7,15 @@ workbox.setConfig({
   debug: false
 });
 
+var authUrl = "<%= Discourse.base_path %>/auth/";
+
 var cacheVersion = "1";
 
 // Cache all GET requests, so Discourse can be used while offline
 workbox.routing.registerRoute(
-  new RegExp('.*?'), // Matches all, GET is implicit
+  function(args) {
+    return !(args.url.origin === location.origin && args.url.pathname.startsWith(authUrl));
```

Comment:
`startsWith` doesn't work in old browsers, but I guess it's not a problem because service workers don't either!